features/build/stibuild.feature

```diff
@@ -135,3 +135,26 @@ Feature: stibuild.feature
     And the "eap-app-1" build completed
     Given 1 pods become ready with labels:
       | application=eap-app |
+
+  # @author xiuwang@redhat.com
+  # @case_id OCP-28891
+  Scenario: Test s2i build in disconnect cluster 
+    Given I have a project
+    When I have an http-git service in the project
+    And I run the :set_env client command with:
+      | resource | dc/git               |
+      | e        | REQUIRE_SERVER_AUTH= |
+      | e        | REQUIRE_GIT_AUTH=    |
+    Then the step should succeed
+    When a pod becomes ready with labels:
+      | deploymentconfig=git |
+      | deployment=git-2     |
+    When I run the :cp client command with:
+      | source | <%= ENV['BUSHSLICER_HOME'] %>/testdata/build/httpd-ex.git | 
```

Comment:
@xiuwang  Could you try to replace `ENV['BUSHSLICER_HOME']` with `BushSlicer::HOME`

Reply:
@liangxia why? any difference?

Reply:
No differences. Just a little shorter, more readable, and with less special characters.

Reply:
@liangxia Updated, and passed in my local host, please help review again. Thanks